### DISCUSSION TITLE
Change font-size of codeblock

### DISCUSSION THREAD

**Discussion by m1rm** [marp-team/marp#519] [Q&A] [answered]
I am using triple backticks in my presentation (markdown file) to define some codeblocks. I would like to change the font-size of the code displayed.

I tried:
```
<style scoped>
    pre {
        font-size: 100px;
    }
</style>
```
based on this [Stack Overflow answer](https://stackoverflow.com/questions/67012975/how-do-i-style-my-code-blocks-in-a-marp-presentation), but it does not work.

The slide does not only hold codeblocks, but also an h2 and normal text.

**What is the selector for the code blocks in marp?**

**Comment by yhatt** [comment, marked as answer]
The tweaked style that you've described should be worked in Marp built-in themes. However, if you are using some customized themes and plugins, it may be using [more specific selector](https://developer.mozilla.org/docs/Web/CSS/Specificity) such as `pre > code` to set the custom font size.

When `pre` selector did not work, you can try to use more specific selector `pre > code`, or `!important` keyword to override the font size.

```css
pre > code {
  font-size: 100px;
}
```

```css
pre {
  font-size: 100px !important;
}
```

**Comment by m1rm** [comment]
Thank you for the reply! In case anybode else stumbles across it: another reason for the font size not increasing in your code blocks seems to be too long lines. If you break them, the text will be displayed with increased font size
